incubator/hnc/api/v1alpha1/hnc_config.go

```diff
@@ -54,9 +54,9 @@ const (
 // TypeSynchronizationSpec defines the desired synchronization state of a specific kind.
 type TypeSynchronizationSpec struct {
 	// API version of the kind defined below. This is used to unambiguously identifies the kind.
-	APIVersion string `json:"apiVersion,omitempty"`
+	APIVersion string `json:"apiVersion"`
```

Comment:
Can you make the same change in `TypeSynchronizationStatus`? Technically it's not necessary since HNC will always fill them in, but we may as well make the code self-documenting.

Reply:
SG, updated! Question: by "self-documenting", did you mean that the it shows the fields are "required" when people read the code?

Reply:
Yup, exactly. It also shows up as "required" in the CRD.